config/settings/base.py

```diff
@@ -237,3 +240,24 @@
 CLIENT_VERIFY_CERTIFICATES = strtobool(
     os.getenv("ROSETTA_CLIENT_VERIFY_CERTIFICATES", "True")
 )
+
+
+SENTRY_DEBUG_URL_ENABLED = False
+if SENTRY_DSN := os.getenv("SENTRY_DSN", ""):
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        environment=os.getenv("SENTRY_ENVIRONMENT", ""),
+        release=get_git_sha(),
```

Comment:
Use the same `BUILD_VERSION` env var as `ds-frontend` here (e.g. https://github.com/nationalarchives/ds-frontend/blob/main/config.py#L16) and set a string similar to https://github.com/nationalarchives/ds-frontend/blob/main/app/__init__.py#L42-L46

Reply:
For release parameter the `BUILD_VERSION` default is blank in ds-frontend. Is this intended, or add `get_git_sha()` as default? Can I have the Readme description for `ds-search`

Reply:
Locally, the `BUILD_VERSION` is blank. It gets populated as part of the CI/CD build process once a build version number is generated:

1. https://github.com/nationalarchives/ds-search/blob/main/.github/workflows/cd.yml#L46-L52, which then calls...
2. https://github.com/nationalarchives/ds-docker-actions/blob/main/.github/actions/docker-build/action.yml#L34